### PR TITLE
$$$$ Cache Money $$$$

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Add country api
Add caching to db calls
Remove libs reference
Re-add model.database
Fix references to models
Update libs pom to remove spring boot refs

### CODE REVIEW

**Comment by sdevalapurkar-bcgov** [comment]
best PR name I've seen in a long time 💯 🥇

File: src/backend/quarkus-management-api/src/main/resources/application.properties [hunk 1]
```diff
@@ -25,3 +25,5 @@ quarkus.http.access-log.enabled=true
 quarkus.http.access-log.pattern=%t "%r" %s %b %D "%{i,Referer}" "%{i,User-Agent}"
 quarkus.http.access-log.log-to-file=false
 
+#Cache Money
```
Comment (sdevalapurkar-bcgov) [inline src/backend/quarkus-management-api/src/main/resources/application.properties:28]: 🤣

**Review by sdevalapurkar-bcgov — APPROVED** [review]
LGTM!

**Comment by TayGov** [comment]
> best PR name I've seen in a long time 💯 🥇

hehe how could I not!

**Comment by sdevalapurkar-bcgov** [comment]
> > best PR name I've seen in a long time 💯 🥇
> 
> hehe how could I not!

6aba4af7bfbc3bd5a21a49ad99fb9746

**Pull request merged**